examples/anomalib_adversary/requirements.txt

```diff
@@ -0,0 +1,2 @@
+anomalib[full] @ git+https://github.com/openvinotoolkit/anomalib.git@v1.0.1
+mart @ git+https://github.com/IntelLabs/MART@main
```

Comment:
Can probably just throw these in setup.py (or ideally pyproject.toml).

Reply:
Thanks for the suggestion. I replaced `setup.py` and `requirements.txt` with `pyproject.toml` in e0117ae8a07d242267e53cb443ff3abcf5b48c69